pages/address/[address]/compare.tsx

```diff
@@ -0,0 +1,122 @@
+/** @jsx jsx */
+import { useEffect, useState } from 'react';
+import { Heading, Box, jsx, Flex, NavLink, Button, Text } from 'theme-ui';
+import { useBreakpointIndex } from '@theme-ui/match-media';
+import ErrorPage from 'next/error';
+import Link from 'next/link';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { Icon } from '@makerdao/dai-ui-icons';
+import { getNetwork } from 'lib/maker';
+import { fetchJson } from 'lib/utils';
+import { useAnalytics } from 'lib/client/analytics/useAnalytics';
+import { ANALYTICS_PAGES } from 'lib/client/analytics/analytics.constants';
+import PrimaryLayout from 'components/layouts/Primary';
+import SidebarLayout from 'components/layouts/Sidebar';
+import Stack from 'components/layouts/Stack';
+import SystemStatsSidebar from 'components/SystemStatsSidebar';
+import ResourceBox from 'components/ResourceBox';
+import PageLoadingPlaceholder from 'components/PageLoadingPlaceholder';
+import useAccountsStore from 'stores/accounts';
+import { usePollVoteCompare } from 'lib/hooks';
+import { AddressApiResponse } from 'modules/address/types/addressApiResponse';
+
+const AddressCompareView = ({ addressInfo }: { addressInfo: AddressApiResponse }) => {
```

Comment:
I don't think we'll want to release this in this sprint but we can leave it for now.